CodeStarterCamp_Week2/Lottery.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Lottery.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by 김예림 on 2023/05/30.
+//
+
+import Foundation
+
+// MARK: - 로또 번호 생성 함수
+func generateLotteryNum() -> Set<Int> {
+    var randomNumSet: Set<Int> = Set()
+    
+    while randomNumSet.count < 6 {
+        let randomNumber = Int.random(in: 1...45)
+        randomNumSet.insert(randomNumber)
+    }
+    
+    return randomNumSet
+}
+
+
+// MARK: - 당첨 번호 확인 함수
+func compareLotteryNum(lottoNum: Set<Int>) {
+    let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
+    var matchNums: [Int] = []
+    
+    for num in lottoNum {
+        if myLottoNumbers.contains(num) {
+            matchNums.append(num)
+            continue
+        }
+    }
+    
+    guard matchNums.isEmpty == false else { return print("아쉽지만 겹치는 번호가 없습니다.")}
+    
```

Comment:
```suggestion
    if matchNums.isEmpty {
        return print("아쉽지만 겹치는 번호가 없습니다.")
    }
```

`만약 matchNums가 비어있다면` 을 코드로 작성했을 때에는 guard 문 보다는 if문이 조금 자연스러운 것 같아요~